### PR TITLE
Task/OPUI-56 angular websockets bower

### PR DESCRIPTION
**Pull request by paul-barry-kenzan**

### CODE REVIEW

File: dist/angular-websocket.min.js [hunk 1]
```diff
@@ -1,2 +0,0 @@
...:\/\//.exec(t);if(!r)throw new Error("Invalid url provided");if("object"==typeof exports&&require)try{s=require("ws"),n=s.Client||s.client||s}catch(i){}return n=n||e.WebSocket||e.MozWebSocket,o?new n(t,o):new n(t)},this.createWebSocketBackend=function(e,o){return t.warn("Deprecated: Please use .create(url, protocols)"),this.create(e,o)}}var o=angular.noop,n=Object.freeze?Object.freeze:o,s=Object.defineProperty,r=angular.isString,i=angular.isFunction,a=angular.isDefined,c=angular.isObject,l=angular.isArray,u=angular.forEach,h=Array.prototype.slice;Array.prototype.indexOf||(Array.prototype.indexOf=function(e){var t=this.length>>>0,o=Number(arguments[1])||0;for(o=0>o?Math.ceil(o):Math.floor(o),0>o&&(o+=t);t>o;o++)if(o in this&&this[o]===e)return o;return-1}),angular.module("ngWebSocket",[]).factory("$websocket",["$rootScope","$q","$timeout","$websocketBackend",e]).factory("WebSocket",["$rootScope","$q","$timeout","WebsocketBackend",e]).service("$websocketBackend",["$window","$log",t]).service("WebSocketBackend",["$window","$log",t]),angular.module("angular-websocket",["ngWebSocket"]),"object"==typeof module&&"function"!=typeof define&&(module.exports=angular.module("ngWebSocket"))}();
```
Comment (obuckley-kenzan) [inline dist/angular-websocket.min.js]: @paul-barry-kenzan 
do we not need a dist file?  what is in _bower.json#main_ for this project?

Comment (paul-barry-kenzan) [inline dist/angular-websocket.min.js]: @obuckley-kenzan 
The output is based on the upstream build. The main file in both `bower.json` and `package.json` point to the root-level `angular-websocket.min.js` file, however, the upstream repo also includes a `dist/angular-websocket.min.js` version so included them to stay in sync (not N*Sync) with upstream. 

https://github.com/AngularClass/angular-websocket

Comment (obuckley-kenzan) [inline dist/angular-websocket.min.js]: So not sure we should delete this file then?  It seems like what we need to do is generate dist.  They have a script for it in the repo
https://github.com/ThomsonReuters-IPS/angular-websocket/blob/master/dist.sh

**Comment by paul-barry-kenzan** [comment]
@obuckley-kenzan 
Ran `npm run dist` and committed generated files.

**Pull request merged**